File: src/main/java/com/sasfmlzr/vkbot/controller/menuprogram/PropertiesProgramWindowController.java
package com.sasfmlzr.vkbot.controller.menuprogram;

import javafx.fxml.FXML;
import javafx.fxml.Initializable;
import javafx.scene.control.TextArea;

import java.net.URL;
import java.util.ResourceBundle;


public class PropertiesProgramWindowController implements Initializable
{
	public final static String resourcePath = "com.sasfmlzr.vkbot.resourcebundle.PropertiesProgramWindow.messages";
	public final static String fxmlPath = "PropertiesProgramWindow.fxml";
	@FXML private TextArea textLog;

	public static String token1,token2,token3,token4;		// токены из ini для ботов
	public static String userId1,userId2,userId3,userId4;	// userid из ini для ботов
	public static String mode1,mode2;	// userid из ini для ботов
	public void initialize(URL location, ResourceBundle resources)
	{




	}

	public void initWindow() {

		textLog.setText("");
		/*
		BufferedReader bReader = new BufferedReader(new FileReader("src/resources/locale/PropertiesProgramWindow/Properties.ini"));
		System.out.println(bReader);
		String s;
		while((s=bReader.readLine())!=null){
			textLog.appendText(s + "\n");
	//		System.out.println(s);
		}
		*/
//////////////логи берутся из файла









	}


}

File: src/main/java/com/sasfmlzr/vkbot/controller/menuprogram/PropertiesProgramWindowController.kt
package com.sasfmlzr.vkbot.controller.menuprogram

import javafx.fxml.FXML
import javafx.fxml.Initializable
import javafx.scene.control.TextArea
import java.io.BufferedReader
import java.io.FileReader
import java.net.URL
import java.util.*

class PropertiesProgramWindowController : Initializable {
    @FXML
    private lateinit var textLog: TextArea

    override fun initialize(location: URL, resources: ResourceBundle) {}

    fun initWindow() {
        val bReader = BufferedReader(FileReader("src/resources/locale/PropertiesProgramWindow/Properties.ini"))
        val text = bReader.readText()
        textLog.text = text
    }

    companion object {
        const val resourcePath = "com.sasfmlzr.vkbot.resourcebundle.PropertiesProgramWindow.messages"
        const val fxmlPath = "PropertiesProgramWindow.fxml"

        var token1: String? = null
        var token2: String? = null
        var token3: String? = null
        var token4: String? = null        // токены из ini для ботов
        var userId1: String? = null
        var userId2: String? = null
        var userId3: String? = null
        var userId4: String? = null    // userid из ini для ботов
        var mode1: String? = null
        var mode2: String? = null    // userid из ini для ботов
    }
}
